web/src/i18n/locales/en.yaml

```diff
@@ -367,13 +366,17 @@
         payers who may benefit must contribute their share from the outset.
         The absence of ancillary payers is not sufficient cause for
         Medicaid to be primary payer.
+
+      reminder: >-
+      Include any participating non-Medicaid FTEs.
```

Comment:
I think this line needs to be indented a bit further to get the CircleCI build to go.

Reply:
Thanks! I'll fix it now.